src/models/experts/asyncActions.ts

```diff
@@ -48,13 +48,59 @@ export const fetchExperts = createAsyncThunk(
       return {
         expertIds: appendIds,
         experts,
-        meta: {
-          pageNumber: response.data.number,
-          totalPages: response.data.totalPages,
-          totalElements: response.data.totalElements,
-          isLastPage: response.data.last,
-          appendExperts,
+        totalPages: response.data.totalPages,
+        totalElements: response.data.totalElements,
+        isLastPage: response.data.last,
+        appendExperts,
+      };
+    } catch (err) {
+      return rejectWithValue(err.response?.data);
+    }
+  },
+);
+
+export const fetchExpertsAutorsList = createAsyncThunk(
+  'experts/loadExpertsAuthorsList',
+  async (options: IExpertsAutorsListOptions, { getState, rejectWithValue }) => {
+    try {
+      const {
+        experts: {
+          data,
+          meta: {
+            size,
+            textFields: { author },
+            sort: { order, sortBy },
+          },
+        },
+      } = getState() as any;
```

Comment:
As far as I can see `getState()` returns an object of `RootStateType`, we use only some of its properties that seems don't have a separate type defined.

Reply:
since most other places use `getState() as any` too, let's keep it this way. But please create a ticket to address this later across all the locations.